MonoGame.Framework/Platform/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -251,6 +252,29 @@ protected virtual void CreateDeviceResources()
 
         internal void CreateSizeDependentResources()
         {
+#if WINDOWS_UAP
```

Comment:
Could you extract this into a function? That'd keep this function a little easier to read.

Reply:
Done.